internal/datastore/spanner/options.go

```diff
@@ -37,14 +37,17 @@ const (
 type Option func(*spannerOptions)
 
 func generateConfig(options []Option) (spannerOptions, error) {
+	// originally SpiceDB didn't use connection pools for Spanner SDK, so it opened 1 single connection
+	// This determines if there are more CPU cores to increase the default number of connections
+	defaultNumberConnections := max(1, math.Round(float64(runtime.GOMAXPROCS(0)/2.0)))
```

Comment:
>Did you set to maxprocs/2 to leave more room for non-spanner work?

Because I don't know if folks are running lots of replicas and this could open a whole lot of connections to Spanner, so tried to be more conservative. I feel like GFE should be able to handle it but 🤷🏻 

>So I would think just setting to GOMAXPROCs is a fine default. (This quote says session pool but from context they mean connection count)

Yep! I agree that's the recommendation, it's just not sure how it's going to affect existing deployments